src/content/lesson/what-are-third-party-libraries.md

```diff
@@ -0,0 +1,244 @@
+
+1. Install Pandas on your computer.
+
+```bash
+pip install pandas
+```
+2. Import Pandas into your working module.
+
+```py
+import pandas as pd
+```
+3. Start enjoying the features of Pandas.
+
+```py
+serie = pd.Series([1, 4, 5, 7, 9])
+
+data_frame = pd.DataFrame({
+    "Nombre": ["MacOS", "Windows", "Linux"],
+    "Color": ["Gris", "Negra", "Red"]
+})
+
+print(serie)
+print(data_frame)
+```
+> output: 
+```bash
+// Serie output
+0    1
+1    4
+2    5
+3    7
+4    9
+dtype: int64
+
+// Dataframe output
+    Nombre  Color
+0    MacOS   Gris
+1  Windows  Negra
+2    Linux    Red
+```
+
+If you want to start your career in data science or artificial intelligence, Pandas will be your ally for data cleaning and analysis. This library allows you to read **.csv files**, **Excel** files, **JSON** files, and more
+
+### Matplotlib
+
+![Example of data visualization in matplotlib](https://miro.medium.com/v2/resize:fit:1400/1*EsqDYFK-IzGEAm4FyZP0wQ.jpeg)
+
+[Matplotlib](https://matplotlib.org/stable/index.html) is a Python library used to create high-quality visualizations and graphs. It provides an interface to produce a variety of charts, from simple to complex, that are essential for effective data communication.
+
+To use Matplotlib in your own projects, follow these steps:
+
+1. Install Matplotlib on your computer.
+
+```bash
+pip install matplotlib
+```
+
+2. Import Matplotlib into your working module.
+
+```py
+import  matplotlib.pyplot  as  plt
+```
+
+3. You can now start using all the features of Matplotlib.
+
+```py
+# Sample data: years and sales
+years = [2015, 2016, 2017, 2018, 2019, 2020]
+sales = [50000, 60000, 75000, 90000, 110000, 130000]
+
+# Create a line chart
+plt.plot(years, sales, marker='o')
+
+# Add labels and title 
+plt.xlabel('Años')
+plt.ylabel('Ventas')
+plt.title('Crecimiento de Ventas a lo largo de los Años')
+
+# Show the chart
+plt.grid(True)
+plt.show()
+```
+
+In this example, we create a mock visualization of the number of sales over the years of a company. Using the `plot()` method, we create the presentation table. Then, with the label and title methods, we create `labels` and the `title` of our table. Finally, we use the `grid()` function to display the grid of our visualization.
+
+## Conclusion
+
+Third-party libraries are an essential tool for any programmer. They provide an efficient way to leverage tested and reliable functionality, speeding up development and improving project quality. Third-party libraries are especially useful in fields like data science or machine learning because they offer a wide range of features that will help you save time and effort when creating your projects. If you want to learn more about the topic, we recommend reading the article [Introduction to Pandas in Python](https://4geeks.com/lesson/intro-to-pandas) on the 4Geeks Blog.
```

Comment:
~If you want to learn more about the topic, we recommend reading the article [Introduction to Pandas in Python](https://4geeks.com/lesson/intro-to-pandas) on the 4Geeks Blog.~